scripts/sign.py

```diff
@@ -67,7 +67,9 @@ def get_args(logger):
         help='Command, one of [' + ', '.join(command_base) + ']')
     parser.add_argument('--uuid', required=True,
                         type=uuid_parse, help='String UUID of the TA')
-    parser.add_argument('--key', required=True,
+    parser.add_argument('--version', type=int_parse, default=0,
+                        help='Decimal version value for the TA, defaults to 0')
```

Comment:
How about inserting "anti-rollback" or "rollback protection" somewhere in this line?